src/main/java/TinhBieuThuc.java

```diff
@@ -0,0 +1,17 @@
+public class TinhBieuThuc {
+
+    public double tinhBieuThuc(int n){
+        double result = 0.0;
+        int mauso = 0;
+        double valueafter = 1;
+
+        do{
+            result += valueafter;
+            mauso++;
+            valueafter *= (double) 1/mauso;
+        }while (mauso <= n);
```

Comment:
Bài này cái vòng lặp chạy từ 1 đến n nên phải dùng vòng for cho dễ hiểu

Reply:
Done